renderdoc/api/replay/vk_pipestate.h

```diff
@@ -1114,6 +1114,13 @@ If the list is empty, fdm_offset is disabled and rendering is as normal.
 )");
   rdcarray<Offset> fragmentDensityOffsets;
 
+  DOCUMENT(R"(If VK_QCOM_multiview_per_view_viewports is enabled, contains whether multiple viewports
+are allowed to be passed in a single view.
+
+If only a single viewport is present in any view, multiviewPerViewViewports is set to false;
+)");
+  bool multiviewPerViewViewports;
```

Comment:
This bool is added to the pipe state here but never actually filled in. Is this left over from an earlier draft, or is there more code to handle this that's missing?

Reply:
Leftover from an earlier draft, I'd thought I would need to add support there but it turns out it was unnecessary. Removed.